test/JDBC/input/doublequoted_string-vu-verify.sql

```diff
@@ -43,15 +43,15 @@ go
 
 set quoted_identifier off
 go
-select "ab""cd"
+select "ab""cd"
 go
-select "de'fg"
+select "de'fg"
 go
-select 'ab"cd'
+select 'ab"cd'
 go
-select "ab'cd"
+select "ab'cd"
 go
-select 'ab"cd'
+select 'ab"cd'

```

Comment:
why is this changes being shown?

Reply:
That's weired , thanks for finding that, the content looks identical and I also didn't intend to do this change that's why I didn't notice there's such diff show in the git diff.